pandas/core/frame.py

```diff
@@ -1964,6 +1965,29 @@ def to_feather(self, path):
 
         to_feather(self, path)
 
+    def to_markdown(self):
```

Comment:
Should `**kwargs` be passed through to tabulate?

Reply:
Sure, done